### PR TITLE
Flux getting started

### PR DESCRIPTION
**Pull request by sanderson**

### CODE REVIEW

File: content/flux/v0.7/_index.md [hunk 1]
```diff
@@ -11,5 +11,9 @@ menu:
 ## [Introduction to Flux](/flux/v0.7/introduction)
 A quick introduction to Flux and its design principles.
 
+## [Enable Flux](/flux/v0.7/introduction/installation)
+Flux is packaged with InfluxDB v1.7+ and does not require any additional installation,
+however it does need to be enabled.
+
 ## [Getting started with Flux](/flux/v0.7/introduction/getting-started)
```
Comment (stevebang) [inline content/flux/v0.7/_index.md]: Change to "Get started with Flux" -- imperative like "Enable Flux" and matches title on Introduction page.

File: content/flux/v0.7/introduction/getting-started/_index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+## What you will need
+
+##### InfluxDB v1.7+
+Flux v0.7 (technical preview) is built into InfluxDB v1.7 and can be used to query data stored in InfluxDB.
+InfluxDB v1.7's `influx` CLI also includes a new `-type=` option that, when set to `flux`, will start an
+interactive Flux Read-Eval-Print-Loop (REPL) allowing you to write and run Flux queries from the command line.
+
+---
+
+_See the [InfluxDB installation instructions](/influxdb/v1.7/introduction/installation)
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/_index.md]: Change to:
"For information about downloading and installing InfluxDB 1.7, see [InfluxDB installation](...)."

File: content/flux/v0.7/introduction/getting-started/_index.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+for information about downloading and installing InfluxDB v1.7._
+
+---
+
+##### Chronograf v1.7+
+**Not required but strongly recommended**.
+Chronograf v1.7's Data Explorer provides a user-interface (UI) for writing Flux scripts and visualizing results.
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/_index.md]: Remove hyphen from user interface.

File: content/flux/v0.7/introduction/getting-started/_index.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+Dashboards in Chronograf v1.7+ also support Flux queries.
+
+---
+
+_See the [Chronograf installation instructions](/chronograf/v1.7/introduction/installation)
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/_index.md]: Change to:
"For information about downloading and installing Chronograf v1.7, see [Chronograf installation](...)."

File: content/flux/v0.7/introduction/getting-started/_index.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+for information about downloading and installing Chronograf v1.7._
+
+---
+
+![Flux Builder](/img/flux/flux-builder.png)
+
+---
+
+_If using Docker, the [InfluxData Sandbox](/platform/installation/sandbox-install)
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/_index.md]: If you are using Docker, ...

File: content/flux/v0.7/introduction/getting-started/_index.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+
+## Key concepts
+Flux introduces important new concepts you should understand as you get started.
+
+### Buckets
+Flux introduces "buckets," a new data storage concept for InfluxDB.
+A **bucket** is a named location where data is stored that has a retention policy.
+It's similar to an InfluxDB v1.x "database," but is a combination of both a database and a retention policy.
+When using multiple retention policies, each is its own bucket.
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/_index.md]: Each what is its own bucket? It sounds like you're saying that each bucket is its own bucket.

File: content/flux/v0.7/introduction/getting-started/_index.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+![Flux in the Data Explorer](/img/flux/flux-builder-start.gif)
+
+This will provide **Schema**, **Script**, and **Functions** panes.
+The Schema pane allows you to explore your data.
+The Script pane is where you write your Flux script.
+The Functions pane provides a list of functions available in your Flux queries.
+
+### 2. The influx CLI
+The `influx` CLI is an interactive shell for querying InfluxDB.
+With InfluxDB v1.7+, use the `-type=flux` option to open a Flux REPL where you write and run Flux queries.
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/_index.md]: With InfluxDB v1.7+, use the `influx` shell command with the `-type=flux` option to open ...
OR
With InfluxDB v1.7+, use the `influx -type=flux` command to open a ...

File: content/flux/v0.7/introduction/getting-started/query-influxdb.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+from(bucket:"telegraf/autogen")
+  |> range(start: -15m)
+  |> filter(fn: (r) =>
+    r._measurement == "cpu" AND
+    r._field == "usage_system" AND
+    r.cpu == "cpu-total")
+  |> yield()
+```
+
+> Chronograf automatically assumes a `yield()` function at the end of each script in order to visualized the data.
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/query-influxdb.md]: Are you trying to say that I don't need to add a yield() function in Chronograf? Or is it a best practice to include it to make the script clearer to follow?

File: content/flux/v0.7/introduction/getting-started/syntax-basics.md [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+description: An introduction to the basic elements of the Flux syntax with real-world application examples.
+menu:
+  flux_0_7:
+    name: Syntax basics
+    parent: get-started
+    weight: 3
+---
+
+
+Flux, at its core, is scripting language, but designed specifically for working with data.
```
Comment (stevebang) [inline content/flux/v0.7/introduction/getting-started/syntax-basics.md]: Flux, at its core, is a scripting language designed specifically for working with data.

File: content/flux/v0.7/introduction/getting-started/_index.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+#### Group Keys
+Every table has a **group key** which describes the contents of the table.
+It's a list of columns for which every row in the table will have the same value.
+Columns with unique values in each row are **not** part of the group key.
+
+As functions process and transform data, each modifies the group keys of output tables.
+Understanding how tables and group keys are modified by functions is key to properly
+shaping your data for the desired output.
+
+###### Example group key
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.7/introduction/getting-started/_index.md]: The h6 formatting matches bold + italicized text which is what I've been using to label code blocks. This is a recent change in the CSS.

**Pull request merged**